### ISSUE TITLE
Create integration test for pulling images with big layers

### ISSUE THREAD

**Issue by olegmoz** [artipie/docker-adapter#153]
In practice layers of Docker images might be huge (hundreds of megabytes or even gigabytes) and we need to support pulling such images from Artipie. We need to have integration test with Docker client downloading huge image from Artipie to see what features we are lacking and keep this functionality covered in future.
Large image pushing test could be found in `LargeImageITCase`. Example of pull tests could be found in `DockerSliceITCase`

**Comment by 0crat** [comment]
@olegmoz[/z](https://www.0crat.com/u/olegmoz) I'm not managing this repo, remove the [webhook](https://github.com/artipie/docker-adapter/settings/hooks) or contact me in Slack, as explained in [§11](http://www.zerocracy.com/policy.html#11); I'm not managing `artipie/docker-adapter` GitHub repository, you have to contact me in Slack first, see [§11](http://www.zerocracy.com/policy.html#11) /cc @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
I'm not managing `artipie/docker-adapter` GitHub repository, you have to contact me in Slack first, see [§11](http://www.zerocracy.com/policy.html#11)

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@cc4a991f-2993-4061-849c-be39b0f7623a, thread: pool-1048-thread-1
com.zerocracy.radars.github.ThrottledComments[74] java.lang.IllegalStateException: Can't post an identical message to artipie/docker-adapter#153: I'm not managing `artipie/docker-adapter` GitHub repository, you have to contact me in Slack first, 
```

[0.53.15](https://github.com/zerocracy/farm): Issue: artipie/docker-adapter#153, Action: `unassigned`

**Comment by olegmoz** [comment]
@olenagerasimova may I assign this issue on you?

**Comment by olenagerasimova** [comment]
@olegmoz sure, but I have to warn you that I will be able to do it only the next week as I have NEO training these Thursday and Friday

**Comment by olegmoz** [comment]
@olenagerasimova that's fine, thanks

**Comment by 0crat** [comment]
I'm not managing `artipie/docker-adapter` GitHub repository, you have to contact me in Slack first, see [§11](http://www.zerocracy.com/policy.html#11)

**Comment by olenagerasimova** [comment]
closed via #246

**Comment by 0crat** [comment]
I'm not managing `artipie/docker-adapter` GitHub repository, you have to contact me in Slack first, see [§11](http://www.zerocracy.com/policy.html#11)
